### PR TITLE
Add Chinese Translation（Test version）

### PR DESCRIPTION
**Pull request by AzakoYin**
Another translation version based on Eustiya's translation version .
I fixed many translation mistakes and try my best to make people who are new to this mod can easily understand.
However, there are a few text-heavy items left that have not been translated.
I will do my best to continue to update in the days to come
At present, this version I uploaded is the best version among all the Chinese translations at present.
Translated by Eustiya    AzakoYin

### CODE REVIEW

**Comment by Speiger** [comment]
Small info from my side @AzakoYin your wiki also undos some of the work done by others.
Translates them back to english.

File: src/main/resources/assets/ic2/lang/en_us_origin.json [hunk 1]
```diff
@@ -0,0 +1,702 @@
+{
```
Comment (Speiger) [inline src/main/resources/assets/ic2/lang/en_us_origin.json:1]: Please remove this file before finishing your translation

File: src/main/resources/assets/ic2/lang/probe_zh_cn.json [hunk 2]
```diff
@@ -134,7 +134,6 @@
 "ic2.probe.teleporter.type.item":"物品",
 "ic2.probe.teleporter.type.fluid":"液体",
 "ic2.probe.teleporter.type.energy":"能量",
-"ic2.probe.teleporter.type.spawner": "刷怪笼",
```
Comment (Speiger) [inline src/main/resources/assets/ic2/lang/probe_zh_cn.json:137]: Why is this removed?

File: src/main/resources/assets/ic2/lang/wiki_zh_cn.json [hunk 3]
```diff
@@ -1,705 +1,705 @@
...
...有 4 种酿造模式：\n- 朗姆酒酿造\n- 啤酒酿造\ n- 威士忌酿造\n- 药水酿造§<newPage>§<header;Rum Brewing;true>\n酿造朗姆酒是最简单的一种。\n只需放入甘蔗，等待其完成即可饮用！ \n\n与其他选项相比，它也相当快，最多几分钟。§<newPage>§<header;Beer Brewing;true>\n酿造啤酒相当简单，\n你加入啤酒花、小麦和水，然后 等一等。\n你可以每人扔 32 个。\n\n- 啤酒花和小麦之间的比例决定酒精度数的强度，\n\n- 而小麦和水之间的比例决定如何 固体饮料是。\n- 水决定产量。\n\na 酒精度数决定效果，\n持续时间由固体比决定，\n强度由酿造质量决定。\n\n最好的酿造质量需要 8 小时。\n10 天后酿造变质 ． 1.4 年。 所以一周等于 10 年。\n当敲击桶时，年份计数会向下舍入到最接近的酿造步骤。\n所以这个不会变坏。§<newPage>§<header;Shaky Effect;true>\nDrinking Whiskey 导致你喝醉。\n那通常意味着什么？ \n直走变得不可能。\n\n这就是摇晃效果发挥作用的地方。§<newPage>§<header;Shaky Effects;true>\n虽然它不会影响你的移动，但它会影响你被击中的能力 由 projectil 的。\n基本上摇晃你的身体以至于“Drunk Foo”成为一个可行的选择。§<newPage>§<header;Potion Brewing;true>\nPotion Brewing 也相当简单。\n\n它就像一个 普通药水架，但不是一次酿造 1 个步骤。\n你使用原料来设置你想要制作的东西，然后让它一次批量处理所有步骤。\n\n\n\n\n例如 你扔进去的抗火药水：\n地狱疣、岩浆膏和水\n然后水开始酿造。\n\n在此之前你可以扔进最多 20 个红石或荧石。\n- §l红石§r：增加持续时间 药水。 最多 30 分钟。\n\n\n- §lGlowstone§r：增加药水的效力。 最高 3 级。\n \n但请记住，每增加一个等级，效力就会减少一半，持续时间也会减半。 \n\n酿造的质量决定实际应用了多少持续时间/效力。§<newPage>§<header;Potion Brew Yields;true>\n最好的酿造大约需要 4 小时。\n\n最大的一批 您一次可以酿造 12 瓶药水，或 120 支药箭。\n\n这将持续足够长的时间，直到下一次酿造完成。§<newPage>§<header;Potion Overbrewing;true>\n注意 20 小时后 在总酿造时间中，当你打开桶时，药水会变质并变成瞬间伤害 4 的云。\n所以不要让它长时间无人看管。\n\n但这涵盖了药水！",
```
Comment (Speiger) [inline src/main/resources/assets/ic2/lang/wiki_zh_cn.json:602]: Why is this entry being translated back to english?

File: src/main/resources/assets/ic2/lang/wiki_zh_cn.json [hunk 4]
```diff
@@ -1,705 +1,705 @@
...
-"wiki.ic2.subpage.colossal_machine.recycler":"巨大回收机",
-"wiki.ic2.subpage.crop.body":"§<newPage>§<header;Crop Stat;true>\n§lGain Stat§r增加了收获产量。\n当人工收获时，这可能是巨大的，因为少数作物会大量掉落。\n自动收获不会获得如此大的产量增加，但仍会大幅增加产量。\n\n当统计值达到24或更高时，将减少额外种子袋的机会。§<newPage>§<header;Crop Stats;true>\n§l抵抗力统计§r增加了对抗杂草的强度。\n在31级时，它不再被杂草所取代。\n但当该统计值超过28时，它参与杂交繁殖的机会就会减少。§<image;ic2:textures/wiki/crops/stats.png;33;20;78>§<padding;2> \n作物受环境影响§<newPage>）§<header;Stat Warning;true>\n这三项统计数据大大提高了作物的生长速度，并确保作物不会挨饿。\n每个生物群落都会根据您所在的位置提供一些好处/缺点。§<newPage>§<header;Growth Stat;true>\n例如，对于湿度来说，黑暗是可怕的，而海洋却为它创造奇迹。\n经验法则是越高越好。§<newPage>§<header;Gain Stat;true>\n养分和湿度都受到其下方农田/底土的影响。\n并且可以根据其所在的生物群落或肥料或水合作用来提高。§<newPage>§<header;Resistance Stat;true>\n作物也需要健康的环境才能更快地生长。\n有三个因素可以提高环境质量。\n-§lSkylight Access§r\n-§l Space§r：周围没有固体块或作物。\n-？l高度§r：在一定程度上，较高的海拔有助于环境质量。§<newPage>§<header;Crop Boosters;true>\n水下作物也需要一个良好的环境。\n这是一个有点棘手的创造。\n有几种方法可以增加它。\n-§lPlay§r：周围土壤中的粘土会稍微增加它；ic2:textures/wiki/crops/environment.png；33;20;78>§<image;ic2:textures/wiki/crops/environment.png;33;20;78> \n作物育种是作物系统的核心。\n而且它可以很直接。\n将2-4种作物放在你想要育种的作物棒上，然后再放上一套作物棒。\n你不能保证育种过程中使用的作物超过2种，但至少使用了2种。§<padding;2>§<newPage>",
```
Comment (Speiger) [inline src/main/resources/assets/ic2/lang/wiki_zh_cn.json:613]: Why is this translated back to english?

File: src/main/resources/assets/ic2/lang/wiki_zh_cn.json [hunk 5]
```diff
@@ -1,705 +1,705 @@
...
-"wiki.ic2.subpage.crop.breeding":"那么繁殖是如何工作的呢？这很简单！几乎每一个组合都可以产生任何§<newPage>§<header;Crop Breeding;true>\n但结果如何决定\n\n它基于一个点系统。\n点是根据作物共有多少个特征或这些特征彼此之间的关系有多密切来生成的。\n这些点的总和与所有亲本作物一起计算，然后从中随机选择一个。\n尽管有1个覆盖。如果亲本作物等于可以采摘的作物，则会获得500点的平buff。\n使其更有可能。\n\n§<redirect;true;Crop Analyzer;ic2:preview.crop_analyzer>可以模拟示例并显示潜在结果。\n\n除此之外，它的随机运气是什么。§<newPage>§<header;Stat Breeding;true>\n现在，每当你培育一种作物时，它也会改变统计数据。\n这是如何做到的，也很简单。\n\n它取父代统计数据的总和，每个统计数据，然后除以父代计数。\n这就是你的平均统计数。§<newPage>§<header;Stat Breeding;true>\n然后添加一些波动，在任意方向随机应用2个统计点，每个父母都在场。\n因此，您可以通过作物育种丢失或获得统计数据。§<newPage>§<header;Crop Copying;true>\n现在获取大量种子真的很困难，也很烦人。\n所以你也可以很简单地复制作物。\n\n基本上，如果育种棒只有一个可能的亲本可以繁殖，它会克隆亲本作物。\n没有统计改变，没有作物改变，直接克隆。§<newPage>§<header;Weed Ex;true>\n§<redirect;true;Weed Ex;ic2:preview.weed_ex>可以保护作物不被杂草破坏。\n但如果一切都很好，过量使用会伤害作物。特别是统计数据。\n因此，经验法则是，仅在weedex几乎为空时使用一次。§<newPage>§<header;Crop Analyzer;true>\n您可以通过使用§<redirect;true;Crop Analyzer;ic2:preview.crop_analyzer>\n\n一般情况下，当您的热库中有作物分析仪时，One Probe会为您显示更多信息！§<newPage>§<header;Farmland / Subsoil!;true>",
```
Comment (Speiger) [inline src/main/resources/assets/ic2/lang/wiki_zh_cn.json:614]: Why is this translated back to english?

File: src/main/resources/assets/ic2/lang/wiki_zh_cn.json [hunk 6]
```diff
@@ -1,705 +1,705 @@
...
-"wiki.ic2.subpage.uu_crop_library.head":"UU作物库",
-"wiki.ic2.subpage.villager.body":"IC2Classic 还在游戏中添加了一些村民。\n这些可以用游戏中的不同方块解锁。§<newPage>§<header;Electric Engineer;true>\n§lBlock:§r §lIron Furnace§r\nThis Electrified Villager 是你的组件供应商，它在销售电路和其他一些东西。\n如果你的 Iron 不足，它会很有用！\n虽然我会避开他，因为一些村民告诉他电击让他发疯了......§ <newPage>§<header;Nuclear Engineer;true>\n§l方块：§r §l核反应堆§r\n发光的绿色村民是你的反应堆供应商，销售反应堆组件和铀...\n这就是发光的原因.. .我想。\n但这是获得大量铀的一种方法。§<newPage>§<header;Crop Farmer;true>\n§lBlock:§r §lSimple Crop Library§r\nFake Farmer 是你的本地人 作物爱好者。 他没有任何可疑之处。\n他出售统计数据不佳或不错的农作物，或一些与农作物相关的物品。§<newPage>§<header;Demolishon Man;true>\n§lBlock:§r §lITNT§r\n尘土飞扬的村民 是你的爆炸工程师。\n需要炸掉什么东西吗？ 他就是那个人！\n出售各种形式的炸药和破碎的金属！\n只要确保永远不要让他独自留在你的基地炸药！§<newPage>§<header;Brewing Master;true>\n§lBlock:§r §lBarrel§r\n醉酒的村民是你的酿造专家！\n销售各种形式的啤酒和威士忌。\n非常好的交谈对象！§<newPage>§<header;Greg;true>\n§lBlock:§r § lPixelfoam GregTech§r\n格雷格是……格雷格。 他太疯狂了，我们不得不把他变成村民！\n他也同意了……嗯，他只说了“hmmmmmmmmmm”，但那应该是同意了……\n对了！ 对吗？？？\n他卖 CFoam、工具和其他一些东西。\n小心点。 有些交易是真实的！§<newPage>§<header;Recipes;true>",
```
Comment (Speiger) [inline src/main/resources/assets/ic2/lang/wiki_zh_cn.json:696]: Why is this translated back to english?

File: src/main/resources/assets/ic2/lang/wiki_zh_cn.json [hunk 7]
```diff
@@ -1,705 +1,705 @@
...
-"wiki.ic2.subpage.villager.head":"村民",
-"wiki.ic2.subpage.villager.recipes":"在这里，您可以找到创建所需块的所有合成表！",
-"wiki.ic2.subpage.villager_o_mat.body":"Villager-O-Mat 是一个自动交易员，它使用身份盗窃来像您一样进行交易！\n\n虽然这在开始时听起来很糟糕，但实际上非常有用。\n因为您保留了交易员将获得的所有有用折扣 随着时间的推移。\n或者，如果你救了他们，救世主会打折。§<newPage>§<header;Family Friendly Game Mechanics!;true>\n让我们忽略你被激励导致需要拯救村民的情况！ \n\n§l\ncough\n§r §l\ncough\n§r De/Zombification 以获得折扣 §l\ncough\n§r §l\ncough\n§r§<newPage>§<header; UI;true>\n§<image;ic2:textures/wiki/villager_o_mat.png;85>\n§<padding;2>\nUI 看似复杂，其实很简单。\n顶部的栏是 存储的 XP，在进行交易时自动收集。§<newPage>§<header;UI;true>\n村民选择器跟踪它可以找到的所有村民\n并为您提供所有村民以供选择。\n同时 向您显示 XP 条。\n\n没有 XP 条的村民是 V 的“类别” 灾厄村民，这使您可以批量选择它们以减少研磨工作。§<newPage>§<header;Trade Selection;true>\n中心部分基本上是您的交易选择器。\n您只需单击一个交易即可禁用/启用它。 \n\nIn / Out 部分基本上是告诉你它从哪里导入用于交易的项目以及它也从哪里导出交易结果。\n输出支持 Tubes，但不支持输入。§<newPage>§<header;How does it work?;true>\nVillager-O-Mat 是一台相当慢的机器。\n它每分钟只尝试交易一次。 但它会尝试一次处理每笔选定的交易。\n如果至少完成了一笔交易，它将向你收取每笔选定交易 6000EU，无论是否交易。§<newPage>§<header;How does it work?;true>\n这听起来真的很贵，但如果您选择了 20 笔交易，这意味着 6000EU，它会降低到实际消耗的 100EU/t。\n这并没有那么贵。\n而且这还不算 因为村民每天只能交易两次。",
```
Comment (Speiger) [inline src/main/resources/assets/ic2/lang/wiki_zh_cn.json:699]: Why is this translated back to english?

File: src/main/resources/assets/ic2/lang/wiki_zh_cn.json [hunk 8]
```diff
@@ -1,705 +1,705 @@
...
-"wiki.ic2.subpage.villager_o_mat.head":"村民交易箱",
```
Comment (Speiger) [inline src/main/resources/assets/ic2/lang/wiki_zh_cn.json:700]: Why is this translated back to english?

File: src/main/resources/assets/ic2/lang/wiki_zh_cn.json [hunk 9]
```diff
@@ -1,705 +1,705 @@
...
...g;90>\n§<padding ;5>\n蓝色叠加层用于更改方块的方向。\n方向根据您单击的一侧设置。\n- §lCenter§r：将方向设置为您单击的一侧。 （如果潜行则相反）\n- §lLeft§r：将方向设置为单击方向的左侧。\n- §lRight§r：将方向设置为单击方向的右侧。\n- §lBottom /Top§r: 将方向设置为向下/向上。\n\n\n因此，如果您没有单击中心，则您将根据要前往的一侧设置方向。§<newPage>§<header; 黄色叠加层；true>\n§<image;ic2:textures/wiki/wrench_yellow.png;90>\n§<padding;5>\n黄色叠加层用于特殊操作。\n例如从管道中移除采矿管道，但 它可能有很多用途。§<newPage>§<header;Red Overlay;true>\n§<image;ic2:textures/wiki/wrench_red.png;110>\n§<padding;5>\n 红色覆盖层用于块移除。 移除机器时，侧面无关紧要。§<newPage>§<header;Wrench Removal;true>\n§<image;ic2:textures/wiki/wrench_loss.png;70>\n§<padding;5>\ n当移除一台机器时，不能保证你能得到准确的块。§<newPage>§<header;Wrench Removal;true>\n相反，如果你不走运，你会得到适当层级的机器块。 \n\n减少机器损失的方法之一是用财富附魔你的扳手。\n每级提供 10% 的乘数，堆叠。\n但粗略的解决方案只能让你到此为止！\n\n另一个 方法是通过 mod 取得进展并获得 §<redirect;true;Electric Wrench;ic2:preview.electric_wrench>，\n它具有 §l\"无损\"§r 模式。\n该模式追溯确保机器是 不会丢失，但会以其非常有限的用途中的 1 次为代价。\n给出一些知识：您使电路过载以实现完美的结果，但电路只能处理这么多。\n稍后还有另一个升级 即 100% mod 内无损。\n\n可以进一步升级，因为它仅在 mod 内无损，而不是所有内容。§<newPage>§<header;Controversy;true>\n此功能一直存在争议，\n有些人真的不喜欢 \"loss\" 功能太多以至于他们超越了 \"Agree to disagree\"。\n\n我对这个功能做了一些更改，因为我喜欢它，以使其更易于访问。\n但有些人仍然不 我不在乎，宁愿放弃这个功能。\n\n引用 TFS Android 16 的话：\n§l\“你的合规性不是一个因素\”§r\n\n所以如果你那么讨厌它，那么 模组不适合你。",
```
Comment (Speiger) [inline src/main/resources/assets/ic2/lang/wiki_zh_cn.json:701]: Why is this translated back to english?

**Comment by Eustiya** [comment]
@Speiger 
His text translation is based on version ic2classic 2.02,therefore, these texts are not added. 
Talk to him and learn about he has not completed all his text translations. 
In the next time, he would upload more new versions and correct some other minor errors.
the wiki translation is very nice,I think it can benefit more players.
I will also try to cooperate with him to make the translation better

**Comment by Eustiya** [comment]
@Speiger 
His text translation is based on version ic2classic 2.02,therefore, these texts are not added. 
Talk to him and learn about he has not completed all his text translations. 
In the next time, he would upload more new versions and correct some other minor errors.
the wiki translation is very nice,I think it can benefit more players.
I will also try to cooperate with him to make the translation better

File: src/main/resources/assets/ic2/lang/probe_zh_cn.json (hunk 2 above)
Comment (AzakoYin) [reply src/main/resources/assets/ic2/lang/probe_zh_cn.json:137]: i didnt delete anything.......   let me check . sorry about that

File: src/main/resources/assets/ic2/lang/wiki_zh_cn.json (hunk 9 above)
Comment (AzakoYin) [reply src/main/resources/assets/ic2/lang/wiki_zh_cn.json:701]: > 

These texts often contain idiomatic nouns that I can't explain clearly, so I'm playing around while translating this mod, so some items are translated back into English. I apologize for this phenomenon. This is my first time working as a translator, so I behave like a newbie in every way and cause you a lot of trouble. Sorry very much! I've closed the pull request.
